bwapi/Library/BWAPILIBUnitTest/UnitTypeTest.cpp

```diff
@@ -14,866 +17,1052 @@
 using namespace BWAPI;
 
 std::string get_unitType_string(const testing::TestParamInfo<UnitType>& value) {
-  std::string result = value.param.getName();
-  if (result.empty()) {
-    result = std::to_string(value.index);
-  }
-  return result;
+	std::string result = value.param.getName();
```

Comment:
Can you change your editor settings:
1. Replace tabs with spaces (automatic in editor).
2. Use an indent width of 2 spaces.

I'll look into enforcing the [style guide](https://github.com/bwapi/bwapi/blob/main/CONTRIBUTING.md) at build time. (except for ANSI style braces)